templates/default/ckan_properties.ini.erb

```diff
@@ -152,7 +152,7 @@ ckan.datarequests.default_organisation = open-data-administration-data-requests
 ckan.odi_certificates.certificate_base_url = https://certificates.theodi.org/en/datasets?
 ckan.odi_certificates.certificate_img_query_parameters = {"datasetUrl":"", "format":"png", "type":"badge"}
 ckan.odi_certificates.certificate_link_query_parameters = {"datasetUrl":""}
-ckan.odi_certificates.dataset_base_url = https://<%= node['datashades']['public_tld'] %>
+ckan.odi_certificates.dataset_base_url = https://<%= @app_url %><% node['datashades']['ckan_web']['endpoint'] %>
```

Comment:
This is supposed to use PROD for all environments. I didn't like the decision, but it was a deliberate decision.